### PR TITLE
[DFFS-4] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by odun-netizen**
image
image

### CODE REVIEW

File: submissions/odunayoFeyiAkinbile/HTML/product-page.html [hunk 1]
```diff
@@ -9,6 +9,8 @@
         <h1>FAME</h1>
     </header>
     <nav>
+        <a href="../html/contact-form.html">Contact Form</a>
+        <a href="../html/product-page.html">Product</a>
```
Comment (Ireti203) [inline submissions/odunayoFeyiAkinbile/HTML/product-page.html:13]: This link wont work. The file is in the HTML folder not 'html'

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/odunayoFeyiAkinbile/HTML/contact-form.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <head>
+        <meta charset="UTF-8">
+        <title>Contact Form</title>
+    </head>
+    <body>
+        <h1>Contact Form</h1>
+        <form action="#" method="POST" enctype="multipart/form-data">
+            <nav>
+                <a href="../html/product-page.html">Product</a>
+                <a href="../html/contact-form.html">Contact Form</a>
```
Comment (Ireti203) [inline submissions/odunayoFeyiAkinbile/HTML/contact-form.html:12]: This link will not work. The folder name is incorrect

**Review by Ireti203 — CHANGES_REQUESTED** [review]
Is there a reason you are creating a new contact-form.html file? You have that previously, you should only add the links.

File: submissions/odunayoFeyiAkinbile/HTML/contact-form.html (hunk 2 above)
Comment (odun-netizen) [reply submissions/odunayoFeyiAkinbile/HTML/contact-form.html:12]: yes when i clone dufuna-fem repo the folder is not there so I create it again

File: submissions/odunayoFeyiAkinbile/HTML/product-page.html (hunk 1 above)
Comment (odun-netizen) [reply submissions/odunayoFeyiAkinbile/HTML/product-page.html]: i have made the changes

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**